contrib/cirrus/runner.sh

```diff
@@ -19,15 +19,17 @@ set -eo pipefail
 # shellcheck source=contrib/cirrus/lib.sh
 source $(dirname $0)/lib.sh
 
+showrun echo "started"
```

Comment:
I was going to suggest using `msg` instead, but never-mind [it doesn't show context](https://github.com/containers/automation/blob/main/common/lib/console_output.sh#L69-L71).  Looking at the code, I believe I stripped it out because it was getting too noisy.  Anyway, would you consider changing this to "starting" for consistency with `setup_environment.sh`?

Reply:
Yeah.... `msg()` was my first thought, and is the correct way to do it, but that lives in a separate repo. Too complicated. I've changed it to "starting".